source/clear-linux/reference/bundles/openssh-server.rst

```diff
@@ -110,9 +110,35 @@ To enable root login via SSH, perform the following steps:
    .. code-block:: bash
 
       mkdir /etc/ssh
+      
+#. Create the following file, if it does not already exist:
+   :file:`/etc/ssh/sshd_config`
+   
+#. Set the configuration variable in /etc/ssh/sshd_config
 
-#. Set the configuration variable.
-
-   .. code-block:: bash
-
-      echo "PermitRootLogin yes" >> /etc/ssh/sshd_config
+    .. code-block:: console
+    
+      PermitRootLogin yes
+      
+Enable X11-forwarding 
+*****************
+ To enable X11-forwarding via SSH, perform the following steps:
```

Comment:
I suggest adding a sentence here after the heading that says what S11 forwarding is useful for and why one might want X11 forwarding. 

It would be good to have this for the root login section too but I understand that was already missing before your changes.

Reply:
added description to enplane